recipe/meta.yaml

```
@@ -15,13 +15,13 @@ package:
 
 source:
   - url: https://scipopt.org/download/release/scipoptsuite-{{ scip_version }}.tgz
-    sha256: a3c1b45220252865d4cedf41d6327b6023608feb360d463f2e68ec4ac41cda06
+    sha256: aa9c5456810d7425d2ad409364b3a069ed3a492331cc7256aae9b64f2591b66c
     folder: scipoptsuite
     patches:
       - 0002-Require-PCRE-on-WIN32.patch
 
 build:
-  number: 7
+  number: 4
```

```suggestion
  number: 0
```
We are starting back to zero with this version update (it should be checked at the PR, not commit level).

---

I see. Thanks for reviewing!